uint8 uav_id
bool hold_on
---
bool status